examples/models/object_detection/faster_rcnn/train.py

```diff
@@ -241,13 +263,13 @@ def pad_fn(examples):
 with strategy.scope():
     # The keras huber loss will sum all losses and divide by BS * N * 4, instead we want divide by BS
     # using NONE reduction as also summing creates issues
-    rpn_reg_loss_fn = tf.keras.losses.Huber(reduction=tf.keras.losses.Reduction.NONE)
+    rpn_reg_loss_fn = tf.keras.losses.Huber(reduction=tf.keras.losses.Reduction.SUM)
```

Comment:
It seems like these loss_fns are no longer used in this script since we're using the defaults from `compile`?

Reply:
Good catch! Removed.